Units/fabricaheks.lua

```diff
@@ -46,7 +46,7 @@ local unitDef = {
 	-- Building
 	
 	Builder = true,
-	canMove = true,

```

Comment:
revert this and i'll merge
this is neede to set a destination point for units begin created